#General Parameters

rate                 : 10       # Loop Rate of the while loop 
publish_mav_odometry : true     # Whether to publish mav_odometry or not
verbose              : false    # Print additional debug and timing information.

#Parameters essential for trajectory generation

visualize            : true     # Whether to visualize trajectory in rviz or not
command              : true     # Send commands to mav
dimension            : 3        # Determines the spatial dimension of trajectory.
v_max                : 2.0      # Maximum velocity of the MAV in the entire trajectory.
a_max                : 2.0      # Maximum acc. of the MAV in the entire trajectory.
distance             : 1.0      # Distance by which to seperate additional markers.